app/controllers/computacenter/api/cap_usage_controller.rb

```diff
@@ -1,5 +1,25 @@
 class Computacenter::API::CapUsageController < Computacenter::API::BaseController
   def bulk_update
     logger.info "bulk_update, params = #{params}"
+    validate_xml!('CapUsage.xsd')
+    @batch = create_batch(@parsed_xml)
+    @batch.process!
+    render status: status_code(@batch)
+  end
+
+private
+
+  def create_batch(parsed_xml)
```

Comment:
```suggestion
  def build_batch(parsed_xml)
```
`create` in ActiveRecord implies persistence to the DB, which this method doesn't do, so `build` may be less likely to be misunderstood.

Reply:
good point, done